Examples/Python/GroomUtils.py

```diff
@@ -177,21 +181,23 @@ def applyResampling(outDir, refFile, inDataListSeg, inDataListImg=[]):
 
     # get reference image
     refImg = Image(refFile)
+    refImg.antialias(antialias_iterations)
 
     for i in range(len(inDataListSeg)):
-        segoutname = rename(inDataListSeg[i], segoutDir, 'resized')
+        segoutname = rename(inDataListSeg[i], segoutDir, 'aligned')
         outSegDataList.append(segoutname)
         if inDataListImg:
-            rawoutname = rename(inDataListImg[i], rawoutDir, 'resized')
+            rawoutname = rename(inDataListImg[i], rawoutDir, 'aligned')
             outRawDataList.append(rawoutname)
 
         # resize images to reference images
         img = Image(inDataListSeg[i])
-        transform = createTransform(Matrix()) # create identity matrix
-        img.resample(transform, refImg.origin(), refImg.dims(), refImg.spacing(), refImg.coordsys(), InterpolationType.NearestNeighbor).write(segoutname)
+        rigidTransform = ImageUtils.createRigidRegistrationTransform(Image(inDataListSeg[i]).antialias(antialias_iterations), refImg, isoValue, icp_iterations)
+        img.applyTransform(rigidTransform, refImg.origin(), refImg.dims(), refImg.spacing(), refImg.coordsys(), InterpolationType.Linear).write(segoutname)
+
         if inDataListImg:
             img = Image(inDataListImg[i])
-            img.resample(transform, refImg.origin(), refImg.dims(), refImg.spacing(), refImg.coordsys(), InterpolationType.NearestNeighbor).write(rawoutname)
+            img.applyTransform(rigidTransform, refImg.origin(), refImg.dims(), refImg.spacing(), refImg.coordsys(), InterpolationType.Linear).write(rawoutname)
```

Comment:
same here.

Reply:
Same here meaning they should be antialiased? These are the raw images not segmentations, do we need to antialias them?

Reply:
no, if they are images, no antialiasing needed.